go.mod

```diff
@@ -73,7 +73,7 @@ require (
 	k8s.io/apiserver v0.22.5
 	k8s.io/client-go v0.22.5
 	k8s.io/component-base v0.22.5
-	k8s.io/cri-api v0.25.0
+	k8s.io/cri-api v0.25.3
```

Comment:
Question; I noticed that `k8s.io/cri-api` was updated to v0.25.3. That's just an update of the version, because there were no changes in vendored files.

However, I started to look "is there an equivalent update in `main` that should be cherry-picked", which led me to https://github.com/containerd/containerd/commit/202abf8fc1928e7517e4235d75a3c8624321e056 (and https://github.com/containerd/containerd/commit/38ad785cef5f81acd09b1ed22472ade504d1c024), and now I noticed that the release/1.6 branch seems to be a bit of a "Franken-version" of kubernetes dependencies; https://github.com/containerd/containerd/blob/61f9fd88f79f081d64d6fa3bb1a0dc71ec870523/go.mod#L71-L78

I always assumed that these versions should (must?) be updated in tandem, so that they all align, or is it "expected" / "OK" for the CRI-API to have a different version?; how problematic is it that they're not aligned?

I also tried what diff showed when I added a `replace k8s.io/cri-api => k8s.io/cri-api v0.22.5`, and that _DID_ introduce a diff in the vendor;

```bash
git diff -- go.mod vendor/k8s.io/cri-api
```

Diff attached; 
[diff.txt](https://github.com/containerd/containerd/files/12657911/diff.txt)